library.rb

```diff
@@ -0,0 +1,116 @@
+require 'json'
+
+class Library
+
+  def to_s
+    "library"
+  end
+
+  def write(data)
+    File.open("storage/json/#{self.to_s}.json","w") do |file|
+      file.write(data.to_json)
+    end
+  end
+
+  def read
+    File.read("storage/json/#{self.to_s}.json")
+  end
+
+  def parseJson
+    JSON.parse(self.read)
+  end
+end
+
+class Author < Library
```

Comment:
Please split your library.rb file into few small files, one class per file. It a good practice to focus just on one class.
Also about inherited.
I think you use it incorrectly here, because it's semantically different things: Author and Library, Book and Library. Inheritance good to use if you see that one class can be more specific than parent class. For example:
Player < Object, Pistol < Weapon.